dist/ngFormBuilder.js

```diff
@@ -2925,6 +2925,9 @@ module.exports = ['debounce', function(debounce) {
         $scope.formComponents = _.cloneDeep(formioComponents.components);
         _.each($scope.formComponents, function(component, key) {
           component.settings.isNew = true;
+          if (!component.settings.hasOwnProperty('isConfigurable')) {
```

Comment:
This is where I default the value to true, if it's not provided by the person registering the component.